inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs

```diff
@@ -0,0 +1,50 @@
+using Epilogue.Actors.Icarasia.States;
+using Godot;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+[Tool]
+public partial class parallax_background_blueprint : ParallaxBackground
+{
+
+	/// <summary>
+	/// This export velue adds the ability to modify the enum directly from the editor
+	/// </summary>
+	string bg;
+	[Export(PropertyHint.Enum, "None,default")]
+	string type_of_bg {
```

Comment:
Please use `camelCase` when naming variables (this variable should be renamed to `typeOfBg`)